scrapy/extensions/feedexport.py

```diff
@@ -241,55 +243,104 @@ def __init__(self, crawler):
 
         self.storages = self._load_components('FEED_STORAGES')
         self.exporters = self._load_components('FEED_EXPORTERS')
+        self.storage_batch_size = self.settings.get('FEED_STORAGE_BATCH_ITEM_COUNT', None)
         for uri, feed in self.feeds.items():
             if not self._storage_supported(uri):
                 raise NotConfigured
+            if not self._batch_deliveries_supported(uri):
```

Comment:
We are not checking if batch deliveries are supported. We are checking if the setting values affecting batch deliveries are consistent. So you can change the method name to something more in that line.

You may not even have to make it specific to batch deliveries, you could give the method a generic name in the line of `_settings_are_valid` or `_options_are_valid`. In the future we can also pass the `feed` variable to that method and use it to check the validity of several options and settings.